app/src/main/AndroidManifest.xml

```diff
@@ -12,6 +12,19 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.MoviO"
-        tools:targetApi="31" />
+        tools:targetApi="31">
+        <activity
```

Comment:
What is this activity for?

Reply:
It was for demo/testing and I forgot to delete it.